kik_unofficial/client.py

```diff
@@ -868,7 +867,8 @@ def data_received(self, data: bytes):
         # Handle empty packets wont get caught by logic below
         if data == b" ":
             self.loop.call_soon_threadsafe(self.api._on_new_data_received, data)
-        elif re.search(b'<pong/>', data):
+        # Sometimes Pong comes back in an ack message, handle those.
```

Comment:
Are there cases where `pong` comes back without an ack message?

Reply:
Yes it can come both ways. I added better handling to this to search specifically for just ack packets.